LWA_EPIC/LWA_EPIC.py

```diff
@@ -742,8 +747,8 @@ def main(self):
                 ishape = (self.ntime_gulp, nchan, nstand, npol)
                 ogulp_size = self.ntime_gulp * self.nchan_out * nstand * self.npol_out * 1  # ci4
                 oshape = (self.ntime_gulp, self.nchan_out, nstand, self.npol_out)
-                self.iring.resize(igulp_size)
-                self.oring.resize(ogulp_size)  # , obuf_size)
+                self.iring.resize(igulp_size, buffer_factor= 8)
+                self.oring.resize(ogulp_size, buffer_factor= 128)  # , obuf_size)
```

Comment:
I think @jaycedowell had some concerns about this. IIRC, this looks like we're making large buffers to account for a slow startup, when really we should be aiming to fix the startup issue instead. I don't have a solution, just flagging this for discussion.

Reply:
Yeah, that `buffer_factor` of 128 is really large.  I had done some simple benchmarking and it looked like the problem was with the `GenerateLocations` call in `MOFFCorrelatorOp.main` being slow.  I don't know if that was all of it but it is a factor.

For solutions we could think about something like compute-and-cache.  The first time a particular frequency range is encountered we take the hit on computing and cache the results to disk.  On subsequent encounters with that frequency setup we use what is on disk.

Reply:
That makes a lot of sense. Perhaps we could also reorder so that even if we need to generate the locations it's done before the buffer is allocated/used, so we don't need to be juggling data while we're still setting up.